.env.yaml

```diff
@@ -0,0 +1,10 @@
+UPR_URL:
+  description: Upr DSN url
+  required: true
+  type: url
+  exmaple: https://username:password@upr.io
+
+UPR_CACHE:
+  description: Upr  cache path. optional - should use in-memory cache if unspecified
```

Comment:
1. Duplicated space
2. The description does not match the definition of this variable